src/compiler/types.ts

```diff
@@ -8463,7 +8463,7 @@ namespace ts {
     export interface ResolvedModuleSpecifierInfo {
         modulePaths: readonly ModulePath[] | undefined;
         moduleSpecifiers: readonly string[] | undefined;
-        isAutoImportable: boolean | undefined;
+        isBlockedByPackageJsonDependencies: boolean | undefined;

```

Comment:
This is not a behavior change, but the property became a misnomer because in nodenext, factors that don't contribute to this property can make the module specifiers not auto-importable.

Reply:
"Blocked" sounds like it means "not auto-importable".  Is that correct and, if so, does that mean that the meaning of the flag has flipped?

Reply:
For other reviewers, yes, the meaning has flipped.